content/go/concepts/map/map.md

```diff
@@ -0,0 +1,167 @@
+---
+Title: 'map'
+Description: 'Map is a built-in data structure used to store data of the same or different data type in key-value pairs.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'map'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+A map is a built-in data structure that is used to store a collection of unordered key-value pairs that can be the same or different data type. It is Go's implementation of a Hash Table, which allows for efficient access, insertion, and deletion.
```

Comment:
This applies to the whole file: You need to insert lines between paragraphs, before and after headers, and before and after code blocks.